src/util/fs.js

```diff
@@ -58,15 +60,23 @@ const futimes: (fd: number, atime: number, mtime: number) => Promise<void> = pro
 const copyFile: (src: string, dest: string, flags: number, data: CopyFileAction) => Promise<void> = fs.copyFile
   ? // Don't use `promisify` to avoid passing  the last, argument `data`, to the native method
     (src, dest, flags, data) =>
-      new Promise((resolve, reject) => fs.copyFile(src, dest, flags, err => (err ? reject(err) : resolve(err))))
+      new Promise((resolve, reject) =>
+        fs.copyFile(src, dest, flags, err => {
+          if (err) {
+            reject(err);
+          } else {
+            fixTimes(0, dest, data).then(() => resolve(err)).catch(ex => reject(ex));
```

Comment:
I'm not sure I understand why the first parameter is `0`. The fd `0` is stdin, right?

Reply:
It was `0` just because it's `falsy`. In the case of not using `copyFile` we already have a file open that we can use, instead of having to re-open the file. If the `fd` is falsey, then `fixTimes` opens the file. I was going to pass `undefined` but then Flow gets mad. I'll see if I can make it more understandable...

Reply:
I think you should be able to use `null`, then in the declaration of `fixTime` just use `fd: ?number` (instead of `: number`). Flow should then accept `=== null` without complaining.